docs/config.ld

```diff
@@ -1,24 +1,27 @@
 local args = ...
+local ldoc = nil
 
+-- luacheck: globals new_type
 
 -- Configuration file for ldoc
 
-project='awesome'
-title='awesome API documentation'
-description='API documentation for awesome, a highly configurable X window manager (version @AWESOME_VERSION@).'
+project='awesome' -- luacheck: globals project
```

Comment:
Thanks a lot for that. But we can't remove it without all distros shipping with 0.26.x first. 

As they say, the best time to plant a tree was 30 years ago, the next best time is now. I wish we had taken over ldoc years ago and dropped the stupid useless sandbox. It would have avoided me all the pain to implement this PR... Now `ldoc` is too zombie to get anything merged.